packages/upset/src/components/Sidebar.tsx

```diff
@@ -53,12 +55,18 @@ export const Sidebar = () => {
     secondAggregateBy !== 'None',
   );
 
+  const [ showImportModal, setShowImportModal] = useState(false);
```

Comment:
```suggestion
  const [ showImportModal, setShowImportModal ] = useState(false);
```

We need to add a linter for this project

Reply:
I thought I had a linter setup for this. We should add a precommit hook to lint files like I do for Trrack. Right now linting relies on having ide extension